native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -149,36 +190,36 @@ EMSCRIPTEN_BINDINGS(spine) {
         .value("Clipping", AttachmentType_Clipping);
 
     //////////////////////////////////////////////////////////////////////////////////////////////////////////////
-    //////////////////////////////////////////////////////////////////////////////////////////////////////////////
-    // class_<Vector>("Vector")
-    //     .constructor<>()
-    //     .constructor<const Vector &>()
-    //     .function("size", &Vector::size);
-
```

Comment:
If we bind `spine::Vector`, then could we remove the conversion between `spine::Vector<T>` and `std::vector<T>`?

Reply:
It will be optimized according to this idea later.